pkg/actuators/machine/machine_scope.go

```diff
@@ -5,15 +5,17 @@ import (
 	"fmt"
 	"strings"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
this import can be clubbed with the below k8s block of imports

Reply:
Updated it